Google.Api.Gax.Grpc.Gcp/DefaultChannelCredentialsCache.cs

```diff
@@ -54,12 +59,22 @@ internal DefaultChannelCredentialsCache(IEnumerable<string> scopes)
                     {
                         appDefaultCredentials = appDefaultCredentials.CreateScoped(_scopes);
                     }
+
+                    if (_useUseJwtAccessWithScopes && appDefaultCredentials.UnderlyingCredential is ServiceAccountCredential)
```

Comment:
Asking here for `_useJwtAccessWithScopes` in order to change the credential makes assumption about what the default is in the credential itself. I'd rather not so we can make changes independently.

If you want to optimize instance creation, then it's better if we check that in `ServiceAccountCredential.WithUseJwtAccessWithScopes`, if the value being received as a parameter is the same as the current value, then we return `this`.

and then:
```suggestion
                    if (appDefaultCredentials.UnderlyingCredential is ServiceAccountCredential serviceCredential)
```
and you can delete the first line inside the if.